x-pack/plugins/ml/server/models/data_recognizer/modules/security_auth/ml/suspicious_login_activity.json

```diff
@@ -1,24 +1,17 @@
 {
-  "description": "Security: Auditbeat - Detect unusually high number of authentication attempts.",
-  "groups": [
-    "security",
-    "auditbeat",
-    "authentication"
-  ],
+  "description": "Security: Authentication - Detect unusually high number of authentication attempts.",
```

Comment:
Nit - the other jobs use third person singular, so this shoud be `Detects` to be consistent.

Reply:
`Detects an unusually high number...` would read better here. Or even `Looks for an unusually high number..` to be consistent with the other job descriptions in this module.